### ISSUE TITLE
Couldn't link with INET project

### ISSUE THREAD

**Issue by buraksenkus** [Unipisa/Simu5G#5]
Hi,
I couldn't make the project because I couldn't link Simu5G with INET project on both IDE and command line.
I can run INET examples properly. I'm following [this guide](http://www.simu5g.org/install.html) for installation.

Here is my folder structure:
- omnetpp-6.0pre12
  - samples 
    - inet4
    - Simu5G-1.2.0

1. When I run **make** command, it gives below error.
    ![image](https://user-images.githubusercontent.com/19664856/131678311-9fecb05f-1559-4ebb-8a4e-bde570935a4d.png)

2. When I change `inet4.3` to `inet4` in below line of Makefile in Simu5G-1.2.0 folder

    ```make
    makefiles:
	    @cd src && opp_makemake --make-so -f --deep -o simu5g -O out -KINET_PROJ=../../inet4.3 -DINET_IMPORT -I. -I$$\(INET_PROJ\)/src -L$$\(INET_PROJ\)/src -lINET$$\(D\)
    
    ```
    and re-run **make makefiles** and **make**, it gives below error.
![image](https://user-images.githubusercontent.com/19664856/131679033-2240f5e0-7abb-46fa-9658-ee71b7c150c3.png)

To sum up, I couldn't figure out how to make the project.

Thanks in advance!

**Comment by giovanninardini** [comment]
Hello,

which version of INET are you using?

Btw, the [install guide](http://www.simu5g.org/install.html) says that you should use OMNeT++ 6.0**pre10** or **pre11**, together with **INET 4.3.2**. 
Also, as you already saw in the Makefile, the INET folder should be named as "inet4.3".

Best regards.
Giovanni

**Comment by krzycho2** [comment]
Hi,
FYI, OMNET++ 6.0 pre11 is available on GitHub https://github.com/omnetpp/omnetpp/releases/tag/omnetpp-6.0pre11

**Comment by buraksenkus** [comment]
Thank you for your answers. I have downloaded OMNET++ 6.0pre11 and INET 4.3.2. Now, this error has gone but I'm getting another error: 
```
./apps/d2dMultihop/TrickleTimerMsg_m.h:62:47: error: ‘any_ptr’ was not declared in this scope
   62 | template<> inline TrickleTimerMsg *fromAnyPtr(any_ptr ptr) { return check_and_cast<TrickleTimerMsg*>(ptr.get<cObject>()); }
```

![image](https://user-images.githubusercontent.com/19664856/131832490-bc50fd53-dd45-48a8-af59-43eed51b9353.png)

Btw, it would be better if you change the **inet4** in "*...tarball next to the **inet4** directory...*" statement of install guide as **inet4.3** in order to avoid confusion.

**Comment by krzycho2** [comment]
@buraksenkus I'm not sure what exactly your issue is. But I also had some problems with installing simu5G beside INET 4.3.2 and I solved them all this way:
- I deleted older versions of OMNET bin folders from PATH
- I store INET4.3 and Simu5G-1.2.0 in exactly same location
- Other issues were solved by running "Clean local" command from GUI
I use Ubuntu 20.04.

**Comment by giovanninardini** [comment]
@krzycho2 might be right. It is probably an issue due to those _m.h files were generated by your previous build with the wrong version of OMNeT++.
From the command line, you can do a "make cleanall" for both INET and Simu5G. 
Then make sure that your PATH variable includes the correct OMNeT++ bin folder, and compile everything again. 

Also, thank you for pointing out the misleading information on the website. We will fix it.

**Comment by buraksenkus** [comment]
Thank you for your helps @giovanninardini and @krzycho2. As you said, problem has been solved by execution `make cleanall` in both INET & Simu5G and then running `. setenv`, `make makefiles`, `make` for both projects respectively. 

Best regards!
